Caretaker home

BrendanHorst opened the pull request:
Added functionality to the caretaker's home page
-See a list of all patients in their group that day
-Will display message instead if they are not assigned to a group
-Each field has a checkbox
-Upon submission all fields will be updated as 1 or 0 to reflect their checked status
-If a patient does not have a schedule for that day yet, the fields will display anyway and a schedule will be created upon submission

BrendanHorst committed:
Started table on caregiverHome and created database connection

BrendanHorst committed:
Added table to caretakerHome that displays all patients' schedules

BrendanHorst committed:
Added checkbox inputs to the table data on caregiverHome

BrendanHorst committed:
Added functionality to form on caregiverHome with update/insert queries

BrendanHorst committed:
Moved POST request code above form creation

This way, the form defaults will be up to date with a submitted request

BrendanHorst committed:
Moved caretaker group code above POST request handler

Also reformatted sql queries and centered the success message in the form

BrendanHorst committed:
Fixed logout resetting schedule table

BrendanHorst committed:
Merge branch 'main' into caretaker-home

Thommond commented on views/home/caregiverHome.php:
```diff
@@ -34,5 +34,256 @@
+            <th>Lunch</th>
+            <th>Dinner</th>
+          </tr>
+          <tr>
```
Moving the tr closer to where you are inserting the data may be best

Thommond reviewed (changes requested):
Moving the tr from line 51 to closer to insertion is all that is needed.

BrendanHorst committed:
Moved creation of <td> lower and properly closed it

BrendanHorst committed:
Merge branch 'main' into caretaker-home

BrendanHorst committed:
Fixed errors regarding multiple users on the caretaker home

- Each row is now displayed correctly
- Success message displays once, rather than equal to the number of users
- Added a missing comma in data.sql
